viewer/src/public/migration/Cognite3DModel.ts

```diff
@@ -133,7 +133,7 @@ export class Cognite3DModel extends THREE.Object3D implements CogniteModelBase {
   }
 
   /**
-   * Used to clean up memory.
+   * Cleans up used resources.
```

Comment:
BTW I haven't found any usage of it in reveal manager only in migration wrapper

cc @larsmoa